shared/message/input/Sensors.proto

```diff
@@ -43,46 +42,37 @@ message Sensors {
         float  voltage          = 12;
         float  temperature      = 13;
     }
-
     message Button {
         uint32 id    = 1;
         bool   value = 2;
     }
-
     message LED {
         uint32 id     = 1;
         uint32 colour = 2;
     }
-
-    message FSR {
-        repeated float value  = 1 [packed = true];
-        vec2           centre = 2;
+    message Foot {
+        bool down = 1;
+        mat4 Hwf  = 2;
     }
-
     google.protobuf.Timestamp timestamp     = 1;
     vec3                      accelerometer = 2;
     vec3                      gyroscope     = 3;
 
-    /// This is the transform from the odometry (world) space to robot space (=Htw). It measures the world in robot
-    /// space.
+    /// This is the transform from the odometry (world) space to robot space
+    /// (=Htw).
     mat4            Htw                = 4;
-    repeated FSR    fsr                = 5;
+    repeated Foot   feet               = 5;
     repeated Servo  servo              = 6;
     repeated Button button             = 7;
     repeated LED    led                = 8;
     float           voltage            = 9;
     float           battery            = 10;
-    vec3            centre_of_pressure = 11;
-    mat2            robot_to_IMU       = 12;
+    repeated mat4   forward_kinematics = 15 [(array_size) = 20];
+    vec4            centre_of_mass     = 17;
+    mat4            Hgt                = 18;
+    mat3            inertial_tensor    = 20;
```

Comment:
I don't know if our convention handles inertia tensors, nor do I know how to use them but it would be good to know what space etc this is measured in